File: src/libs/ThirdParty.VBSuppliers/Properties/Travelgate/Thirdparties/TravelgateDerbysoft/TravelgateDerbysoftUOR.vb
Imports ThirdParty.Abstractions.Constants
Imports ThirdParty.Abstractions.Lookups

Public Class TravelgateDerbysoftUOR
    Inherits Travelgate

    Public Sub New(settings As ITravelgateSettings, support As ITPSupport)
        MyBase.New(settings, support)
    End Sub

    Public Overrides Property Source As String
        Get
            Return ThirdParties.TRAVELGATEDERBYSOFTUOR
        End Get
        Set
        End Set
    End Property

End Class

File: src/libs/ThirdParty.CSSuppliers/Travelgate/Thirdparties/TravelgateDerbysoft/TravelgateDerbysoftUOR.cs
namespace ThirdParty.CSSuppliers.Travelgate
{
    using System.Net.Http;
    using Intuitive.Helpers.Security;
    using Intuitive.Helpers.Serialization;
    using Microsoft.Extensions.Logging;
    using ThirdParty.Constants;
    using ThirdParty.Lookups;

    public class TravelgateDerbysoftUOR : Travelgate
    {
        public TravelgateDerbysoftUOR(
            ITravelgateDerbysoftUORSettings settings,
            ITPSupport support,
            HttpClient httpClient,
            ISecretKeeper secretKeeper,
            ISerializer serializer,
            ILogger<TravelgateDerbysoftUOR> logger)
            : base(settings, support, httpClient, secretKeeper, serializer, logger)
        {
        }

        public override string Source => ThirdParties.TRAVELGATEDERBYSOFTUOR;
    }
}